src/main/java/ch/njol/skript/util/Timespan.java

```diff
@@ -32,10 +33,6 @@
 import ch.njol.util.coll.CollectionUtils;
 import ch.njol.yggdrasil.YggdrasilSerializable;
 
-/**
- * @author Peter Güttinger
- * @edited by Mirreducki. Increased maximum timespan.
- */
 public class Timespan implements YggdrasilSerializable, Comparable<Timespan> { // REMIND unit
 
 	private final static Noun m_tick = new Noun("time.tick");
```

Comment:
```suggestion
	private static final Noun TICK_NOUN = new Noun("time.tick");
```

Reply:
There are multiples, this is not a worry